images/pytorch-cuda12/tests/main.tf

```diff
@@ -9,6 +9,7 @@ variable "digest" {
   description = "The image digest to run tests over."
 }
 
+/*
```

Comment:
what did the tooling team say? did we highlight and escalate the issue there first before disabling?

Reply:
We started with disabling just the tests for troubleshooting the issue, but it looks like helm setup is the issue.

Reply:
chatted offline, tests are passing when ran outside CI, so this will allow us to move forward until resolved in CI